charts/ozone/templates/scm/scm-statefulset.yaml

```diff
@@ -31,6 +31,7 @@ metadata:
     app.kubernetes.io/component: scm
 spec:
   replicas: {{ .Values.scm.replicas }}
+  podManagementPolicy: Parallel
```

Comment:
Is a `Parallel` policy disruptive when we use `kubectl scale` up/down. 
It would be interesting to see if the Ratis rings are disrupted. Perhaps a `PreStop` hook for graceful shutdown of OM/SCM/Datanodes is required in a new Jira.

Reply:
I see a different problem: If we use `kubectl scale` the configuration from helpers of cluster ids and so on is also not correct. Puhhh no idea how to solve this at the moment...